.github/workflows/tag.yml

```diff
@@ -0,0 +1,58 @@
+---
+name: "Release a tag"
+on:
+  release:
+    types: [created]
+jobs:
+  release-darwin-amd64:
+    name: release darwin/amd64
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.15.0'
+      - run: go build
+      - run: echo ::set-output name=RELEASE_VERSION::${GITHUB_REF#refs/*/}
+        id: get_tag_name
+      - run: tar -czf "sqlvet-${{ steps.get_tag_name.outputs.RELEASE_VERSION }}-darwin-amd64.tar.gz" "sqlvet";
+      - name: Get release
+        id: get_release
+        uses: bruceadams/get-release@v1.2.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Upload binary
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.get_release.outputs.upload_url }}
```

Comment:
shouldn't we be able to get the upload_url through the github action event directly? Something like this: `github.event.release.upload_url`.

Reply:
Hi, it seems that GitHub didn't put it in the event. I wasn't able to get it and I checked how plugin did it.
By the way I added -ldflags '-w -extldflags "-static"' on build for linux version to make it static and avoir linux dependencies. For example I wasn't able to make it work in a docker alpine image before.